registry/registry.go

```diff
@@ -195,6 +195,46 @@ type Registry struct {
 	views   map[string]*View
 }
 
+// GetViewsAffectedByPath returns all the views in the registry that have visibility
+// into a storage path.
+func (r *Registry) GetViewsAffectedByPath(path string) []*View {
+	var views []*View
+	for _, view := range r.views {
+		for _, rule := range view.rules {
+			if pathChangeAffects(path, rule.originalStorage) {
+				views = append(views, view)
+				break
+			}
+		}
+	}
+
+	return views
+}
+
+func pathChangeAffects(modified, affected string) bool {
+	affectedPathKeys, moddedPathKeys := strings.Split(affected, "."), strings.Split(modified, ".")
```

Comment:
nitpick: I got a bit confused because these are in the reverse order than the arguments, could you swap them?

Reply:
done, thank you